src/lib/ChatRoomManager.ts

```diff
@@ -15,12 +15,12 @@ import {
 } from 'unique-names-generator'
 import { shake256 } from 'js-sha3'
 
-const ROOM_PREFIX: string = 'streamr-chat/room'
-const LOCAL_STORAGE_KEY = 'streamr-chat-rooms'
+const RoomPrefix: string = 'streamr-chat/room'
+const LocalStorageKey = 'streamr-chat-rooms'
 
```

Comment:
Hm, I got a mixed feeling about pascal-casing unscoped consts but let's give it a try if you like it. Feels more natural for enums, heh. :)